packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -223,6 +223,17 @@ exports.mapTagsCategoriesToTaxonomies = entities =>
     return e
   })
 
+exports.mapElementsToParent = entities => entities.map(e => {
+    if (e.wordpress_parent) {
+      // Replace wordpress_parent with a link to the parent node of type.
+      e.parent_element___NODE = entities
+        .filter(el => el.__type === e.__type)
+        .find(t => t.wordpress_id === e.wordpress_parent).id
```

Comment:
I don't think we need to create temporary array with entities filtered by types and we could just add that check in `find`? Unless we would store filtered entities by type for future use so we don't generate this every time (we could do that with `_.groupBy` early or do this on demand with some helper function). What do You think?

Also you create `parent_element` field so probably would need to adjust comment above and not delete `wordpress_parent` field?

Sorry for nitpicking here :)